### PR TITLE
Modify Vagrant file to support 'dev' and 'stage' envs

### LINKED ISSUE #114: Modify Vagrant file to support 'dev' and 'stage

### PR DESCRIPTION
**Pull request by IlanZuckerman**
fix #114

### CODE REVIEW

**Comment by lioramilbaum** [comment]
What is the status of this effort?

File: Vagrantfile [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+end
+
 Vagrant.configure("2") do |config|
-  config.vm.box = "bento/ubuntu-18.04"
   config.vm.provision "shell", path: "bootstrap.sh"
-  config.vm.network "forwarded_port", guest: 5000, host: 5000
+
+  config.vm.define "dev" do |dev|
+    dev.vm.box = "bento/ubuntu-18.04"
+    config.vm.network "forwarded_port", guest: 5000, host: 5000
```
Comment (lioramilbaum) [inline Vagrantfile:17]: s/config/dev/

File: Vagrantfile [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+  end
+
+  config.vm.define "stage" do |stage|
+
+    stage.vm.box = "dummy"
+    stage.vm.box_url = "https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box"
+
+    stage.vm.provider :aws do |aws,override|
+    aws.keypair_name = "aws_key_pair"
```
Comment (lioramilbaum) [inline Vagrantfile:26]: the indentation is wrong

File: Vagrantfile (hunk 1 above)
Comment (IlanZuckerman) [reply Vagrantfile]: +!

File: Vagrantfile (hunk 2 above)
Comment (IlanZuckerman) [reply Vagrantfile]: +1

**Pull request merged**